en/telemetry/telemetry_wifi.md

```diff
@@ -9,4 +9,8 @@ PX4 supports telemetry via UDP and Wifi. It broadcasts a heartbeat to port 14550
 Compatible WiFi Telemetry modules include:
 * [ESP8266 WiFi Module](../telemetry/esp8266_wifi_module.md)
 * [ESP32 WiFi Module](../telemetry/esp32_wifi_module.md)
-* [3DR Telemetry Wifi](../telemetry/3dr_telemetry_wifi.md) (Discontinued)
\ No newline at end of file
+* [3DR Telemetry Wifi](../telemetry/3dr_telemetry_wifi.md) (Discontinued)
+
+# WiFi Telemetry settings for virtual machines
```

Comment:
Thanks. I agree this is important but it is irrelevant to 99% of users, so a distraction here.

Can we move this to https://docs.px4.io/main/en/dev_setup/dev_env_windows_vm.html#windows-virtual-machine-hosted-toolchain ?

Reply:
Sure that's a good idea, it would help to keep all virtual machine-related information on one page which I would actually prefer for myself. Should I close this PR and start a new one on the other page?

Reply:
That would be easiest. I'll close it. Thank you .